### PR TITLE
Breath first graph

### PR DESCRIPTION
**Pull request by ygao0719**
- [x] Top-level README “Table of Contents” is updated
 - [x] Feature tasks for this challenge are completed
 - [x] Unit tests written and passing
     - [x] “Happy Path” - Expected outcome
     - [x] Expected failure
     - [x] Edge Case (if applicable/obvious)
 - [x] README for this challenge is complete
     - [x] Summary, Description, Approach & Efficiency, Solution
     - [x] Link to code
     - [x] Picture of whiteboard

### CODE REVIEW

File: 401-code-challenges/Data-Structures/src/main/java/graph/Graph.java [hunk 1]
```diff
@@ -50,4 +50,30 @@ public int size(){
         return nodes.size();
     }
 
+    public List<Integer> BFS(Node node){
+        List<Integer> order = new ArrayList<>();
+        boolean visited[] = new boolean[nodes.size()];
```
Comment (evanslaton) [inline 401-code-challenges/Data-Structures/src/main/java/graph/Graph.java:55]: You first `[]` are in the wrong place.This should be:
`boolean[] visited = new boolean[nodes.size()];`
